.github/workflows/ci.yml

```diff
@@ -5,6 +5,46 @@ on:
   pull_request:
     branches: [ master ]
 jobs:
+  infra:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: microsoft/playwright-github-action@v1
+    - name: Set up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Get Node PKG Cache dir
+      id: node-pkg-cache
+      run: |
+        node -e "const path = require('path'); const fs = require('fs'); const cachePath = path.join(os.tmpdir(), 'pkg-cache'); fs.mkdirSync(cachePath); console.log('::set-output name=dir::' + cachePath)"
+    - name: Cache driver
+      id: cache-primes
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.node-pkg-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r local-requirements.txt
+        pip install .
+    - name: Lint
+      run: |
+        black --check .
+        mypy .
+        flake8 playwright tests
+    - name: Build driver
+      run: python build_driver.py
```

Comment:
Do we even need the driver here?

Reply:
Definitely not 👍 first thought was to add the publishing later to that job. But maybe it's better to do the publishing in a separate job.